transcendence-app/seeds/003_seed_games.ts

```diff
@@ -1,20 +1,28 @@
 import { Knex } from 'knex';
-import { faker } from '@faker-js/faker';
 import { defaultUsernames } from './utils';
+import { Game } from '../src/game/infrastructure/db/game.entity';
 
-const GAMES_NUMBER = 30;
+const GAMES_NUMBER = 100;
+
+let startTime = Date.now() - 5 * 24 * 60 * 60 * 1000;
 
 const createRandomGame = (userName1: string, userName2: string) => {
-  if (userName1 === userName2) return;
-  else
-    return {
-      playerOneUsername: userName1,
-      playerTwoUsername: userName2,
-      gameDurationInSeconds: faker.random.numeric(),
-      playerOneScore: faker.random.numeric(),
-      playerTwoScore: faker.random.numeric(),
-      gameMode: 'classic',
-    };
+  let losingScore = Math.floor(Math.random() * 10);
+  const key: keyof Game =
+    losingScore % 2 === 0 ? 'playerOneScore' : 'playerTwoScore';
+  const otherKey: keyof Game =
+    key === 'playerTwoScore' ? 'playerOneScore' : 'playerTwoScore';
+  const oldDate = startTime;
+  startTime = startTime + 120 * 1000;
+  return {
+    createdAt: new Date(oldDate),
+    playerOneUsername: userName1,
+    playerTwoUsername: userName2,
+    gameDurationInSeconds: Math.abs(Math.floor(Math.random() * 100)) + 10,
+    [key]: 10,
```

Comment:
we need consistent games, with non overlapping timestamps, and realistic scores, in order to generate meaningful statistics seeds from them